### PR TITLE
Android Filter Support

### PR DESCRIPTION
**Pull request by renganatha10**
Hi This PR contains filter support for android using this [package](https://github.com/MasayukiSuda/ExoPlayerFilter)

And also the option to save the video with selected filter change using this [package](https://github.com/MasayukiSuda/Mp4Composer-android)

### CODE REVIEW

**Comment by CHaNGeTe** [comment]
Thank you so much for the PR @renganatha10 !
Can an usage example be added to the readme with correct output path?

**Comment by renganatha10** [comment]
@CHaNGeTe I have Updated ReadMe for saving video as well some extra step that needed on installation

inputUrl and outputUrl can be a local filesystem url for save option

**Comment by CHaNGeTe** [comment]
Thanks for the updates @renganatha10 !!

File: android-exoplayer/build.gradle [hunk 1]
```diff
@@ -22,11 +22,15 @@ android {
 }
 
 dependencies {
-    implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
+    implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
```
Comment (n1ru4l) [inline android-exoplayer/build.gradle:25]: Can you please remove this whitespace?

**Review by n1ru4l — COMMENTED** [review]
Is it possible to make this optional? This adds two new android dependencies. Not everyone wants to use filters.

On iOS we already do this by using subspecs. Is there something similar for Gradle?

**Comment by CHaNGeTe** [comment]
I agree with @n1ru4l, is it possible?
I have found this: https://docs.gradle.org/current/userguide/feature_variants.html

**Comment by renganatha10** [comment]
@n1ru4l @CHaNGeTe I am not sure how to implement it. Needed help if subspecs can be implemented via Gradle

**Comment by billdami** [comment]
Any updates on this PR?

**Comment by maknom66** [comment]
I tried to use this PR, however i can't see the filter being applied, am i missing something ?

**Comment by sahilshah668** [comment]
any update..?? can we use filters now..???

**Comment by hueniverse** [comment]
Closing this due to lack of activity. If someone wants to rebase it and figure out how to make it optional, I'll revisit.

**Comment by Space6188** [comment]
How to use this pr?

**Pull request closed without merging**